specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -2799,6 +2799,13 @@
             "type": "string",
             "description": "The name of the virtual machine."
           },
+          {
+            "name": "forceDeletion",
```

Comment:
What is the difference in behavior?  The description should make it clear.  A while back there was a thread about enabling optional force delete on VMs when a customer was deleting a resource group.  Is this change related to that?  Was that approach ever finalized?

Reply:
sent you a mail..